src/child_process.ml

```diff
@@ -9,4 +9,25 @@ let exec cmd cb =
 external execSync : string -> Buffer.t = "" [@@bs.module "child_process"]
 
 let execSync cmd =
-  Buffer.toString (execSync cmd) 
+  Buffer.toString (execSync cmd)
+
+type execFileOptions =
+  {
+    cwd: string[@bs.optional ];
+    env: string Js.Dict.t[@bs.optional ];
+    encoding: string[@bs.optional ];
+    timeout: int[@bs.optional ];
+    maxBuffer: int[@bs.optional ];
```

Comment:
Prolly doesn't apply here but I've started to use `float` each time I suspect a large int might be used (for instance file sizes). Both `int` and `float` compile to JS "Number" (bleh!) but `int` declarations can sometime overflow...